readme.md

```diff
@@ -191,6 +240,18 @@ Execute a file synchronously.
 
 Returns or throws a [`childProcessResult`](#childProcessResult).
 
+### \$(options?)\`command\`
+
+Same as `execa()` except both file and arguments are specified in a single tagged template string. For example, ``$`echo unicorns` ``is the same as `execa('echo', ['unicorns'])`.
```

Comment:
We should have some information that any string or array of strings (not sure whether number arguments need be documented?) in `${...}` is automatically escaped.